pkg/server/status.go

```diff
@@ -2349,6 +2349,21 @@ func (h varsHandler) handleVars(w http.ResponseWriter, r *http.Request) {
 	telemetry.Inc(telemetryPrometheusVars)
 }
 
+// Ranges returns current tenant's range info for the specified node.
+func (t *statusServer) Ranges(
```

Comment:
Behavior of this endpoint in external-process mode would be undefined for `local`.

Reply:
Can you clarify this comment? I'm not sure I understand. Is it invalid to request this with `node: local` in the request? If so, should that be documented in the docstring?

Reply:
I've clarified in the comment.

Reply:
And just out of curiosity, I wanted to see where tenant connector picks the KV node and which one. Here is the code, and it appears it picks randomly.
https://github.com/cockroachdb/cockroach/blob/8a35ea7bcbe09740893b004b137faa2f7b801587/pkg/kv/kvclient/kvtenant/connector.go#L1010-L1031